src/rest/payloads/packs.ts

```diff
@@ -1,13 +1,23 @@
+import { Card } from "./index";
+
 /**
  * Payloads for the packs
  */
 export type Pack = {
     packid:number,
     name:string,
-    image:string,
-    contents:number[]
+    image:string
 }
 
 export type PackResult = {
-    cards:number[]
-}
\ No newline at end of file
+    cards:Card[]
+}
+
+export type PackCardSlot = {
+    rarityWeight: {
+        rarity: number,
+        weight: number
+    }[]
+}
+
```

Comment:
this will only ever get used by the sharp wrapper but its fine to leave here

Reply:
I think I will delete it, it feels kinda odd leaving it there
is this mean this PackCardSlot payload could be deleted?

Reply:
> this will only ever get used by the sharp wrapper but its fine to leave here

I think I will delete it, it feels kinda odd leaving it there
is this mean this PackCardSlot payload could be deleted?

Reply:
yeah PackCardSlot is only needed if you are creating packs so you can remove the payload